beaker_kernel/cli/project.py

```diff
@@ -155,12 +163,68 @@ def new_project(ctx: click.Context, name, no_interact, dependencies, no_context)
         result = subprocess.call(args, env=environ)
         if result > 0:
             raise click.ClickException("There was an error setting up your new Beaker project. Please check the output above.")
+    click.echo("Done.\n")
+
+    if not no_interact:
+        if click.confirm("Would you like to install the new project in dev mode in your current environment?", default=False):
+            click.echo("")
+            ctx.invoke(update, path=(Path.cwd() / name))
+        click.echo("")
+
+    click.echo( "Note:")
+    click.echo( "  Some changes, such as adding or moving a context require updating/reinstalling the project.")
+    click.echo(f"  You may need to run 'beaker project update' if you encounter issues after making updates to the project.")
 
 
 # @project.command()
-# @click.argument("project_name", required=False, default=None)
-# def info(project_name):
+# @click.option(
+#     "-e", "--editable", "editable",
+#     is_flag=True,
+#     default=False,
+#     help='Install the project in editable mode (i.e. setuptools "develop mode").',
+# )
+# @click.argument("path", required=False, default=None, type=click.Path(path_type=Path))
+# def install(path):
 #     """
-#     Information about a project. Looks for a project in current location if no project name is provided.
+#     Installs a project
 #     """
-#     pass
+#     pyproject_path = find_pyproject_file(path)
+#     if not pyproject_path:
+#         search_path = path.absolute() if path else Path.cwd()
+#         raise click.ClickException(f"{search_path} does not appear to be part of a valid project.")
+#     project = Project(pyproject_path)
+#     click.echo(f"Reinstalling {project.metadata.core.name} from directory {project.root}")
+#     subprocess.call(
+#         args=[sys.executable, '-m', 'pip', 'install', '-e', '.'],
+#         env=os.environ,
+#         cwd=project.root,
+#     )
+
+
+@project.command()
+@click.argument("path", required=False, default=None, type=click.Path(path_type=Path))
+def update(path):
+    """
+    (Re)installs the project in editable mode (i.e. setuptools "develop mode") in the current python environment.
+
+    """
+    pyproject_path = find_pyproject_file(path)
+    if not pyproject_path:
+        search_path = path.absolute() if path else Path.cwd()
+        raise click.ClickException(f"{search_path} does not appear to be part of a valid project.")
+    project = Project(pyproject_path)
+    click.echo(f"Reinstalling {project.metadata.core.name} from directory {Path(project.root).absolute()}")
+    subprocess.call(
+        args=[sys.executable, '-m', 'pip', 'install', '-e', '.'],
+        env=os.environ,
+        cwd=project.root,
+    )
+
+
+@project.command()
+@click.argument("project_name", required=False, default=None)
+def info(project_name):
```

Comment:
What kind of things do you think would end up going in here?

Reply:
Oh yeah. Thanks for catching that I missed this. I'll go ahead and knock it out real quick. At lest get the basics in.
Obviously, plan to have description, list of contexts and/or subkernels that are included, and hopefully details about features of the contexts/subkernels.